assets/css/app.scss

```diff
@@ -1,36 +1,19 @@
 /* This file is for your main application css. */
-@import "./phoenix.css";
+@import "tailwindcss/base";
+@import "tailwindcss/components";
+@import "tailwindcss/utilities";
 
-/* Alerts and form errors */
-.alert {
-  padding: 15px;
-  margin-bottom: 20px;
-  border: 1px solid transparent;
-  border-radius: 4px;
-}
-.alert-info {
-  color: #31708f;
-  background-color: #d9edf7;
-  border-color: #bce8f1;
-}
-.alert-warning {
-  color: #8a6d3b;
-  background-color: #fcf8e3;
-  border-color: #faebcc;
-}
-.alert-danger {
-  color: #a94442;
-  background-color: #f2dede;
-  border-color: #ebccd1;
-}
-.alert p {
-  margin-bottom: 0;
-}
-.alert:empty {
+@import "base/buttons";
+@import "base/forms";
+
+@import "components/alert";
+@import "components/lead-content";
+
+@import "layouts/app";
+@import "layouts/landing";
+
+@import "screens/dashboard";
+
+iframe {
```

Comment:
This is for the hidden `iframe` for live reloads. 
Tailwind [overrides](https://github.com/tailwindlabs/tailwindcss/blob/master/src/plugins/css/preflight.css#L221) the default hidden behavior with `display: block;` inspired by CSSremedy.

We're using `display: none;` to hide it again.

Reply:
Using `iframe` without the class or id name may be a bit dangerous as it applies to all iframe 😀 . Does the `live reload iframe has some id or class or other attributes` that we can use along with `iframe` to identify it's a `live reload iframe` not the other iframe @bterone

Reply:
They have a `hidden` tag that we can select. FYI, the `iframe` only displays in `development` mode. In production, it doesn't display. 😄 

Fixed in 7afce9070900c4d7859ba0080483577668cc847e